terraform/modules/tf_stream_alert_kinesis_events/main.tf

```diff
@@ -1,33 +1,41 @@
 // AWS Lambda Function Policy
 resource "aws_iam_role_policy" "stream_alert_lambda_kinesis" {
-  name = "stream_alert_lambda_kinesis_${var.role_policy_prefix}"
+  name = "KinesisGetRecords"
   role = "${var.lambda_role_id}"
 
-  policy = <<EOF
-{
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-      "Effect": "Allow",
-      "Action": [
-        "kinesis:DescribeStream",
-        "kinesis:GetRecords",
-        "kinesis:GetShardIterator",
-        "kinesis:ListStreams",
-        "logs:CreateLogGroup",
```

Comment:
Want to confirm that `logs:*` actions are not required for kinesis read? I see these actions are removed from `aws_iam_policy_document`.

Reply:
Yeah it has nothing to do with Kinesis, it was just technical debt